src/main/java/vazkii/pillar/StructureLoader.java

```diff
@@ -126,5 +127,41 @@ public static String getStructureNBTLocation(String jsonFileName) {
 	public static String jsonifySchema(StructureSchema schema) {
 		return gson.toJson(schema);
 	}
-	
+
+	public static StructureSchema getDefaultSchema() {
```

Comment:
By "you" do you mean the end user? (ie: force them to go set something so it gets read) Or do you mean for me to set it to something while writing the file?